plugins/src/ios/OptimoveSDKPlugin.swift

```diff
@@ -20,7 +20,26 @@ import NotificationCenter
             return nil
         }
 
-        let config = OptimoveConfigBuilder(optimoveCredentials: configValues[optimoveCredentialsKey], optimobileCredentials: configValues[optimoveMobileCredentialsKey])
+        let optimoveCredentials = nil
+        if let val = configValues[optimoveCredentialsKey] {
+            if (!val.isEmpty){
+                optimoveCredentials = val;
+            }
+        }
+
+        let optimobileCredentials = nil
+            if let val = configValues[optimoveMobileCredentialsKey] {
+            if (!val.isEmpty){
+                optimobileCredentials = val;
+            }
+        }
+
+        if (optimoveCredentials == nil && optimobileCredentials == nil) {
+            print("Invalid credentials! Please provide at least one set of credentials")
+            return nil;
+        }
```

Comment:
Do we need this guard, I think the native SDK would also throw/crash/abort if both sets are nil.

Or do we want to support this in a 'warning' fashion, i.e. can inject empty creds is a valid use case?

Reply:
ah, right, i wanted to crash, but cannot throw from swift's variable initializer. Native crashing is good for me